Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterOAuthRequest.cs

```diff
@@ -58,17 +59,17 @@ internal class TwitterOAuthRequest
         /// <returns>awaitable task</returns>
         public async Task ExecuteGetStreamAsync(Uri requestUri, TwitterOAuthTokens tokens, TwitterStreamCallbacks.RawJsonCallback callback)
         {
-            using (var request = new HttpHelperRequest(requestUri, HttpMethod.Get))
+            using (var request = new HttpRequestMessage(HttpMethod.Get, requestUri))
             {
                 var requestBuilder = new TwitterOAuthRequestBuilder(requestUri, tokens);
 
-                request.Headers.Authorization = HttpCredentialsHeaderValue.Parse(requestBuilder.AuthorizationHeader);
+                request.Headers.Authorization = AuthenticationHeaderValue.Parse(requestBuilder.AuthorizationHeader);
 
-                using (var response = await HttpHelper.Instance.GetInputStreamAsync(request).ConfigureAwait(false))
+                using (var response = await client.SendAsync(request).ConfigureAwait(false))
```

Comment:
They have 2 modes, which is defined [here](https://msdn.microsoft.com/en-us/library/system.net.http.httpcompletionoption(v=vs.110).aspx). `GetStreamAsync` is currently finished at when the header is read, but the rest of them are finished when the content is read (default value). 

I haven't had a chance to test out Twitter yet but the rest looks okay, even with `StreamsHelper`